packages/smithy-client/src/emitWarningIfUnsupportedVersion.ts

```diff
@@ -0,0 +1,21 @@
+// Stores whether the warning was already emitted.
+let warningEmitted = false;
+
+/**
+ * Emits warning if the provided Node.js version string is pending deprecation.
+ *
+ * @param {string} version - The Node.js version string.
+ */
+export const emitWarningIfUnsupportedVersion = (version: string) => {
```

Comment:
Make `version` optionable? So in client you can go `emitWarningIfUnsupportedVersion(process?.version)`

Reply:
Process will always have version, as it was introduced in `v0.1.3`

Documentation: https://nodejs.org/api/process.html#process_process_version

Reply:
Nvm, I was thinking of other node-like runtime. But then I realize their process API must be Node-compatible--[deno](https://doc.deno.land/builtin/stable#Deno.version) and node-red